### PR TITLE
more olive fixes

### PR DESCRIPTION
**Pull request by regisb**
- Caddyfile as runtime config, not build-time
- Fix authn MFE header logo (#99)
- Enable caddy logging
- Modify all dev image names
- clarify instructions to set the MFE "version"

### CODE REVIEW

File: tutormfe/plugin.py [hunk 1]
```diff
@@ -98,7 +98,7 @@
 # Build, pull and push {mfe}-dev images
 for mfe in ALL_MFES:
     name = f"{mfe}-dev"
-    tag = "{{ DOCKER_REGISTRY }}overhangio/" + mfe + "-dev:{{ MFE_VERSION }}"
+    tag = "{{ DOCKER_REGISTRY }}overhangio/openedx-" + mfe + "-dev:{{ MFE_VERSION }}"
```
Comment (arbrandes) [inline tutormfe/plugin.py:101]: We need to update `local-docker-compose-dev-services` too.

File: tutormfe/patches/openedx-lms-development-settings [hunk 2]
```diff
@@ -41,6 +41,7 @@ MFE_CONFIG = {
     "LMS_BASE_URL": "http://{{ LMS_HOST }}:8000",
     "LOGIN_URL": "http://{{ LMS_HOST }}:8000/login",
     "LOGO_URL": "http://{{ LMS_HOST }}:8000/theming/asset/images/logo.png",
+    "LOGO_WHITE_URL": "http://{{ LMS_HOST }}:8000/theming/asset/images/logo.png",
```
Comment (arbrandes) [inline tutormfe/patches/openedx-lms-development-settings:44]: Nice catch!  I was worried this was a much worse problem.

**Review by arbrandes — CHANGES_REQUESTED** [review]

File: tutormfe/plugin.py (hunk 1 above)
Comment (regisb) [reply tutormfe/plugin.py]: Good catch. I wrote a function so that I stop forgetting about that image.

**Comment by regisb** [comment]
@arbrandes I snuck another commit to document the use of the mfe "version".

File: README.rst [hunk 3]
```diff
@@ -119,7 +121,7 @@ Other Tutor plugin developers can take advantage of this plugin to deploy their
         }
     }
 
-The MFE assets will then be bundled in the "mfe" Docker image and served at ``http(s)://{{ MFE_HOST }}/{{ MYMFE_MFE_APP["name"] }}``. Developers are free to add extra template patches to their plugins, as usual: for instance LMS setting patches to make sure that the LMS correctly connects to the MFEs.
+The MFE assets will then be bundled in the "mfe" Docker image whenever it is rebuild with `tutor images build mfe`. Assets will be served at ``http(s)://{{ MFE_HOST }}/{{ MYMFE_MFE_APP["name"] }}``. Developers are free to add extra template patches to their plugins, as usual: for instance LMS setting patches to make sure that the LMS correctly connects to the MFEs.
```
Comment (arbrandes) [inline README.rst:124]:
```suggestion
The MFE assets will then be bundled in the "mfe" Docker image whenever it is rebuilt with `tutor images build mfe`. Assets will be served at ``http(s)://{{ MFE_HOST }}/{{ MYMFE_MFE_APP["name"] }}``. Developers are free to add extra template patches to their plugins, as usual: for instance LMS setting patches to make sure that the LMS correctly connects to the MFEs.
```

File: tutormfe/plugin.py [hunk 4]
```diff
@@ -96,9 +96,17 @@
 )
 
 # Build, pull and push {mfe}-dev images
+def mfe_dev_docker_image(mfe: str) -> str:
+    return "{{ DOCKER_REGISTRY }}overhangio/openedx-" + mfe + "-dev:{{ MFE_VERSION }}"
+
+
+tutor_hooks.Filters.ENV_TEMPLATE_FILTERS.add_item(
```
Comment (arbrandes) [inline tutormfe/plugin.py:103]: Nice!  I wasn't even aware there was an easy way to do this!

**Pull request merged**